docs/community.md

```diff
@@ -0,0 +1,45 @@
+# Community Resources
+
+We're focused on our community and pay special attention to the
+business use cases they represent.
+We're also eager to hear your feedback and suggestions for this open
+source project.
+
+
+## Getting Help
+
+Links for open source community resources:
```

Comment:
I've just created a `kglab` tag on StackOverflow https://stackoverflow.com/questions/tagged/kglab
You need a certain amount of points to do this, and I have those 😈, so I've created it just in case.
Not sure if it's worth linking yet here.

The interesting thing with SO tags is that you can turn them into a RSS feed e.g. https://stackoverflow.com/feeds/tag?tagnames=kglab and then use the slack RSS app to send notifications into a dedicated slack channel.

You can also use their limited grammar and create an RSS feed with all the libraries you maintain, e.g. https://stackoverflow.com/feeds/tag?tagnames=kglab+or+pytextrank

Reply:
Thank you so much @louisguitton !!
Just added to the Derwen Slack